ow_faults/include/ow_faults/JointsFaults.h

```diff
@@ -0,0 +1,40 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+#ifndef LinkForcePlugin_h
+#define LinkForcePlugin_h
+
+#include <gazebo/common/Plugin.hh>
+
+namespace gazebo
+{
+class JointsFaults : public ModelPlugin
+{
+public:
+  JointsFaults();
+  ~JointsFaults();
+
+  virtual void Load(physics::ModelPtr model, sdf::ElementPtr /* _sdf */);
+
+private:
+  void OnUpdate();
+
+  physics::ModelPtr m_model;
+  double m_AntennaTiltLowerLimit;
```

Comment:
"Antenna" could be "antenna" to be consistent with the rest of our code. Though, looking at the rules, I don't think they specify that this is required. I'm not sure if it matters. Any opinions?

Reply:
you are right, will change

Reply:
addressed